articles/iot-hub/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringV2.md

```diff
@@ -0,0 +1,190 @@
+
+
+  > [!NOTE]
+  > If you happen to close the pane, you can reopen it. Use `Ctrl+Shift+P` (macOS: `Cmd+Shift+P`) to open the command palette, type **Arduino**, and then find and select **Arduino: Examples**.
+
+## Add a new physical device
+
+In the portal, go to **Devices** section and there, click in the  **+New Device** button. 
+
+![Adding a new device](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-add-device.png)
+
+The *new device form* should be filled in.
+1. Click **Physical** in the *Devce type* section.
+2. Define your own Device ID (for example *MXChip* or *AZ3166*).
+3. Choose **Auto generate keys** in the *Autehtication key* section.
+4. Click *Apply* button.
+
+![Adding a new device form](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-add-new-device-form.png)
+
+Wait until the portal finishes the provisioning of the new device.
+
+![Provisioning a new device ](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-add-device-provisioning.png)
+
+
+Then the configuration of the new device, will be shown.
+Copy the **Connection String** generated.
+
+![Device Connection String](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-new-device-connstring.png)
+
+
+This connection string will be used in the next section.
+
+
+
+
+
+## Build and upload the device code
+
+Go back to the Visual Studio Code: 
+
+1. Use `Ctrl+P` (macOS: `Cmd + P`) and type **task config-device-connection**.
+
+![choose your Azure Subscription and your IoT Hub](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/iot-suite-task-config-device-conexion.png)
+
+
+2. The terminal asks you whether you want to use connection string of iot device would you like to use. Select *create new*, and now paste it.
+
+![paste connection string](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/iot-suite-task-config-device-conexion-choose-iot-hub-press-button-A.png)
+
+> [!NOTE]
+> The connection string should be saved in your clipboard if you followed the last section of this tutorial. If not, you should go to the Azure portal and look for the IoT Hub of your Remote Monitoring resource group. There, you can see the IoT Hub connected devices and copy the Device connection string.
+
+
+![look for the connection string](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-connection-string-of-a-device.png)
+
+
+
+3. The terminal sometimes prompts you to enter configuration mode. To do so, hold down button A, then push and release the reset button and then release the button A. 
+The screen displays the DevKit ID and 'Configuration'.
+
+![Device DevKit Screen](media/iot-hub-arduino-iot-devkit-az3166-devkit-remote-monitoringv2/azure-iot-suite-devkit-screen.png)
+
+

```

Comment:
The Build and Upload part is missing:

4. Use Ctrl+Shift+P (macOS: Cmd+Shift+P) to open the command palette, type Arduino, and then find and select Arduino: Upload.